css/main.scss

```diff
@@ -435,6 +435,37 @@ i:hover {
   }
 }
 
+.cookie-banner {
+  width: 100%;
+  padding: 20px 0;
+  position: fixed;
+  bottom: 0;
+  z-index: 9999;
+  background-color: #e7f2ef;
+
+  p {
+    margin: 0;
+  }
+
+  a {
+    color: #688f85;
+  }
+}
+
+.cookie-button {
+  display: inline-block;
+  border-width: 2px;
+  border-style: solid;
+  padding: 5px 50px;
+  cursor: pointer;
+
+  &:hover {
+    background-color: #688f85;
+    color: #e7f2ef;
+    text-decoration: none;
+  }
+}
```

Comment:
Let's reuse the style of the existing buttons on the page: 
<img width="239" alt="bildschirmfoto 2018-01-24 um 14 37 30" src="https://user-images.githubusercontent.com/1510/35334897-25ef3d22-0114-11e8-8849-2f7ee66b19d3.png">